### ISSUE TITLE
[Feature Request] Linux download!?

### ISSUE THREAD

**Issue by Drallas** [yuliskov/SmartTube#3010]
Thanks for the great player. To be able to ditch Android TV completely, it would be nice to have Linux version of this SmartTube. Will there ever by one!?

**Comment by ACR-Jeff** [comment]
There is already a good alternative for Linux on github "FreeTube" https://github.com/FreeTubeApp/FreeTube

**Comment by Drallas** [comment]
Freetube is a privacy YouTube Player, I doesn’t offer login with accounts, hence no watchlist etc…

**Comment by ACR-Jeff** [comment]
> Freetube is a privacy YouTube Player, I doesn’t offer login with accounts, unfortunately no watchlist etc…

Not sure what you're talking about with the watchlist, I exported my YT subscriptions and imported in Freetube without a problem. I then have Freetube setup to load subscriptions at startup just like I would STN, I also have playlists and history, In the settings menu of Freetube you can configure it pretty nice.

**Comment by 9acca9** [comment]
It is not the same, because with SmartTube you can give up thumb to the videos (which is good for the user channel(the creator of content)) and probably when you see a video that is counted  (again for the content creator).
Of course i could use just the ublock and etc. but, it will be nice having an app like smarttube but for linux.

**Comment by JakeRoxs** [comment]
+1 for a linux docker app or something of the like, SmartTube works amazingly on my Shield Pro (it's crazy how much better then the native YT app it is...)

I am currently using FreeTube and it's nice how much less resources it takes for having extra options and such, however as 9acca9 mentioned, it's not the same, no login means no ability to watch "age restricted" videos, importing things is a bit of a PITA and obviously I still do things in my normal YT account and want those to be automatically synced like what is available in SmartTube.

Regardless, appreciate the app, it's great!

**Comment by user12257** [comment]
Freetube does not support remotes really, so I think there is not really a good option for a Linux TV-box.

**Comment by PhillyMay** [comment]
Yeah, a linux port would be great. Android is getting more and more restricted by google. Be it the paths a file manager can access or (at least on the phone side) the ability to install whatever software you want to use. AndroidTV/GoogleTV is loosing money, so maybe they want to close it even more to monetize it better. Also, Amazon moves away from android tv to its own linux os called "vega OS" with initially no Support for sideloading. 

So in the long Term a Switch to linux tv boxes may be necessary. Then it would be cool, if smarttube will be ready. KDE Plasma is getting a revive for its tv Interface. And smarttube would also work greatly on steam os Devices, like the steamdeck or the Lenovo legion Go.

**Comment by taborgreat** [comment]
Seems to me like the world is waiting for someone to design an ad free fully functional YouTube player for Linux systems

**Comment by ACR-Jeff** [comment]
I get it, I want a Linux version as well, Just at this time there isn't one, So we have to go with the options we have, And Freetube is that option right now or YT website.
